xml/System/DateTime.xml

```diff
@@ -118,7 +118,7 @@ This section contains topics for many common uses of the `DateTime` struct:
  [DateTime values and calendars](#datetime-values-and-calendars)   
  [Persisting DateTime values](#persisting-datetime-values)   
  [DateTime vs. TimeSpan](#datetime-vs-timespan)   
- [Comparing for equality within tolerance](#comparing-for-equality-within-tolerance)
+ [Comparing for equality within tolerance](#comparing-for-equality-within-tolerance)   
```

Comment:
nit: should we use an actual list instead, as we've been doing with the see also section to avoid the two spaces at the end?

Reply:
Yes. Definitely, @mairaw